src/main/java/duke/Momo.java

```diff
@@ -0,0 +1,105 @@
+package duke;
+
+import duke.task.Task;
+
+import java.io.IOException;
+import java.time.format.DateTimeParseException;
+
+import duke.ui.TextUi;
+
+public class Momo {
+
+    private Storage storage;
+    private TaskList tasks;
+
+    /**
+     * Creates a Momo with an original task list if file path exists.
+     * Creates a Momo with empty task list else.
+     *
+     * @param filePath the path of the *.txt file including information of existing tasks.
+     */
+    public Momo(String filePath) {
+        storage = new Storage(filePath);
+        try {
+            tasks = new TaskList(storage.load());
+        } catch (ParseException | IOException e) {
+            //String s = Ui.showLoadingError();
+            tasks = new TaskList();
+        }
+    }
+
+
+    /**
+     * Get responses from Momo using the input of users
+     *
+     * @param input input of user
+     * @return answer of Momo
+     */
+    public String getResponse(String input) {
```

Comment:
Do you think is it possible to break this code into 2? Makes it easier to troubleshoot and for future reuse. :)